### PR TITLE
Removes Hulk

### PR DESCRIPTION
**Pull request by Bluedazzled**
## About The Pull Request

Removes the Hulk mutation from the game

## Why It's Good For The Game

Hulk is and always was overpowered, no amount of nerfs or rebalances could fix it. So why not remove it?

## Changelog
:cl:
del: Removed hulk, huzzah!
/:cl:

### CODE REVIEW

File: code/modules/spells/spell_types/wizard.dm [hunk 1]
```diff
@@ -49,7 +49,7 @@
 	range = -1
 	include_user = TRUE
 
-	mutations = list(LASEREYES, HULK)
+	mutations = list(LASEREYES)
```
Comment (JohnFulpWillard) [inline code/modules/spells/spell_types/wizard.dm:52]: The description here needs updating

File: code/modules/antagonists/wishgranter/wishgranter.dm [hunk 2]
```diff
@@ -23,7 +23,6 @@
 	var/mob/living/carbon/human/H = owner.current
 	if(!istype(H))
 		return
-	H.dna.add_mutation(HULK)
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/wishgranter/wishgranter.dm:26]: Will this be getting replaced or will it just be nerfed?

**Comment by Cyberboss** [comment]
i ded

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Hulk injectors on some maps

**Comment by Bluedazzled** [comment]
> Hulk injectors on some maps

Which maps?

**Comment by Cyberboss** [comment]
See the output of the `Compile Maps` action

**Comment by dragomagol** [comment]
#56116 society

**Comment by Cyberboss** [comment]
Yeah, let's not rehash Monday on a Friday

**Comment by Jared-Fogle** [comment]
im getting deja vu

**Comment by D4C-420** [comment]
Hulk ain't that good. It's just stuns are a great and easy way to kill someone.

**Comment by PKPenguin321** [comment]
this might be one of the single most attempted PRs of all time

**Pull request closed without merging**